include/GoogleSync/GoogleSync.php

```diff
@@ -1143,22 +1301,25 @@ public function syncAllUsers()
     {
 
         // First we populate the array of syncable users
-        $this->setSyncUsers();
+        if (!$this->setSyncUsers()) {
```

Comment:
setSyncUser() can throws an exception if retriving a User is implossible. Please catch/handle the error.

Reply:
Done